ui/src/literature/components/LiteratureItem.jsx

```diff
@@ -101,9 +111,13 @@ class LiteratureItem extends Component {
             wrapperClassName="di"
             publicationInfo={publicationInfo}
           />
-          {this.renderBulletIfPublicationInfoAndEprintsNotEmpty()}
+          {this.renderBulletIfPublicationInfoAndConferenceInfoNotEmpty()}
```

Comment:
Now that this has more than 2 items, can you do something like https://github.com/inspirehep/inspirehep/blob/master/ui/src/literature/components/ReferenceItem/ReferenceItem.jsx#L86 instead.

Btw here we can see limitation of `InlineList` and approach of passing the `data` along with `renderItem` instead of directly markup. But that's a later refactor.